RevenueCatUI/Data/Strings.swift

```diff
@@ -128,6 +134,21 @@ extension Strings: CustomStringConvertible {
             "No StoreKit restore purchases logic will be performed by RevenueCat. " +
             "You must have initialized your `PaywallView` appropriately."
 
+        case .no_price_format_priceFormatter_unavailable:
+            return "Could not determine price format because priceFormatter is unavailable."
+
+        case .no_price_format_priceString_incompatible:
+            return "Could not determine price format because priceString is incompatible."
+
+        case .no_price_round_priceFormatter_nil:
+            return "Could not round price because priceFormatter is nil."
+
+        case .no_price_round_priceString_incompatible:
+            return "Could not round price because priceString is incompatible."
```

Comment:
Nit: I'd remove the camel case variable styling naming from the enum and the message 

```suggestion
        case .no_price_format_price_formatter_unavailable:
            return "Could not determine price format because price formatter is unavailable."

        case .no_price_format_price_string_incompatible:
            return "Could not determine price format because price string is incompatible."

        case .no_price_round_price_formatter_nil:
            return "Could not round price because price formatter is nil."

        case .no_price_round_price_string_incompatible:
            return "Could not round price because priceString is incompatible."
```

Reply:
I made the change! Not a big deal, but I find using camel case for these "tokens" makes the message clearer because it directs your attention more precisely to the cause by referring to the variable by name. (The downside being it's now more tightly coupled to the code.)